src/renderer/components/ProgramChange.tsx

```diff
@@ -42,6 +42,10 @@ export function ProgramChange(props: {
   useEffect(updateAccount, [updateAccount]);
   useInterval(updateAccount, 666);
 
+  if (!change) {
```

Comment:
talk about maddening. this shouldn't work, but it does. 
its also a compile error, but it runs (if you make the change while already running.

Reply:
ah, `return null` was what i needed to know